### DISCUSSION TITLE
How can i remove a notification by id?

### DISCUSSION THREAD

**Discussion by Kiriakos41** [thudugala/Plugin.LocalNotification#347] [Q&A] [answered]
I want after i add a notification to center if i want it to remove / cancel it via id somehow?

**Comment by BrianJRobinson** [comment, marked as answer]
// notifId is the Identifier of the Notification that was created.
 LocalNotificationCenter.Current.Cancel(new int[] { notifId }); works for me in Android, but seems to do nothing in iOS 16.2

**Reply by Kiriakos41** [reply]
Thanks!
